### ISSUE TITLE
The networkPluginName parameter does't work on kubeedge 1.12.1

### ISSUE THREAD

**Issue by bluven** [kubeedge/kubeedge#4574]
**What happened**:
I was trying to let FabEdge work with kubeedge 1.12.1 and FabEdge need configure networkPluginName, but my configuration didn't work, edgecore kept reporting: 
```
Jan 30 09:27:42 edge1 edgecore[8274]: I0130 09:27:42.917174    8274 docker_sandbox.go:401] "Failed to read pod IP from plugin/docker" err="Couldn't find network status for default/net-tool-edge1 through plugin: invalid network status for"
Jan 30 09:27:42 edge1 edgecore[8274]: I0130 09:27:42.917682    8274 pod_container_deletor.go:79] "Container not found in pod's containers" containerID="fb0ef73fa32422136eeccca04810b9837b2ef412b74ddfb1a9658cfcaa93a963"
Jan 30 09:27:43 edge1 edgecore[8274]: I0130 09:27:43.924203    8274 docker_sandbox.go:401] "Failed to read pod IP from plugin/docker" err="Couldn't find network status for default/net-tool-edge1 through plugin: invalid network status for"
```
I checked https://github.com/kubeedge/kubeedge/blob/v1.12.1/edge/pkg/edged/config/config.go#L41 and found that ConvertConfigEdgedFlagToConfigKubeletFlag didn't copy networkPluginName to kubeletFlags.

I also checked the code of kubeedge 1.13.0 and found that it pass networkPluginName to kubeletFlags, is there any plan to improve ConvertConfigEdgedFlagToConfigKubeletFlag of kubeedge 1.12.x?

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- KubeEdge version(e.g. `cloudcore --version` and `edgecore --version`):
- <details><summary>Cloud nodes Environment:</summary>

  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>
- <details><summary>Edge nodes Environment:</summary>

  - edgecore version (e.g. `edgecore --version`):
  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>

**Comment by vincentgoat** [comment]
Yes. This fix should be cherry-picked to branch release-1.12. cc @Shelley-BaoYue
